install/resources_generator.py

```diff
@@ -586,7 +601,7 @@ def _validate_ips(ip):
                 quit(1)
 
         def _validate_netmask(netmask):
-            if not netmask.isdecimal() or int(netmask) not in range(0, 33):
+            if not netmask.isdecimal() or int(netmask) not in range(0, 129):
```

Comment:
Shouldn't we check if it's IPv4 or IPv6, and range-check 33/129 appropriately?    This would/could say an invalid IPv4 netmask is valid...

Reply:
@IdoWeka ??

Reply:
I'm not Ido, but IMO, yes, that's more sensible.

Also the minimum is wrong: The minimum netmask length for an IPv6 ULA address is 8 (effectively, it's more like 48 but I am less sure about whether we should actively enforce that).

The minimum (useful) length of an IPv4 netmask is 8 as well, I guess.

Reply:
I'm wondering if he's using the `netmask.isdecimal()` is true for IPv4 (255.255.255.0 would be decimal, I suppose?) and false for IPv6?    I'm don't know what `int(netmask)` would look like.

This could use a comment or two in the code...